src/couch/src/couch_doc.erl

```diff
@@ -270,6 +270,9 @@ transfer_fields([{<<"_replication_state">>, _} = Field | Rest],
 transfer_fields([{<<"_replication_state_time">>, _} = Field | Rest],
     #doc{body=Fields} = Doc) ->
     transfer_fields(Rest, Doc#doc{body=[Field|Fields]});
+transfer_fields([{<<"_replication_start_time">>, _} = Field | Rest],
```

Comment:
Paul had a good idea. Remove the _replication_start_time and for failure case just use state time as start time. And for completed add start time (or rather duration) to `_replication_stats`.

@garrensmith Will this this affect Fauxton code ^ ?